bin/improver-blend-adjacent-points

```diff
@@ -37,7 +37,7 @@ from cf_units import Unit
 from improver.blending.blend_across_adjacent_points import \
     TriangularWeightedBlendAcrossAdjacentPoints
 from improver.argparser import ArgParser
-from improver.utilities.cube_manipulation import merge_cubes
+from improver.utilities.cube_manipulation import MergeCubes
```

Comment:
Why are you choosing to use the class rather than the merge_cubes wrapper here?

Reply:
With `merge_cubes` I've only wrapped the default functionality, which is required for the majority of use cases.  But for time blending I want to use the `check_time_bounds_ranges` argument.  So I need to invoke the class to use that.

My rationale for that design is that it makes the code clearer to see where something is being done differently, as in this case.

Reply:
Ok, that's fine.